### PR TITLE
Use github for gnulib

### PR DESCRIPTION
**Pull request by sylvestre**

### CODE REVIEW

**Comment by BenWiederhake** [comment]
We should leave a comment or something to explain why this replacement is a good idea. Sure, today github.com seems to be vastly faster and more reliable than savannah.gnu.org, but when that changes (or if anyone reads the code) it would be nice to be told *why* we substitute a different repository (even if it *should* contain the same stuff).

File: .github/workflows/GnuTests.yml [hunk 1]
```diff
@@ -75,7 +75,15 @@ jobs:
         repository: 'coreutils/coreutils'
         path: '${{ steps.vars.outputs.path_GNU }}'
         ref: ${{ steps.vars.outputs.repo_GNU_ref }}
-        submodules: recursive
+        submodules: false
+
+    - name: Override submodule URL and initialize submodules
+      run: |
+        git submodule sync --recursive
+        git config submodule.gnulib.url https://github.com/coreutils/gnulib.git
```
[the remark this answers is not in the document]
Comment (sylvestre) [reply .github/workflows/GnuTests.yml:83]: nope, it adds 

```
[submodule "gnulib"]
	url = https://github.com/coreutils/gnulib.git
```
in .git/config

**Comment by BenWiederhake** [comment]
Yay, no more timeouts!

**Comment by sylvestre** [comment]
and less load for the GNU project :)

**Pull request merged**